hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieWriteClient.java

```diff
@@ -1637,6 +1637,8 @@ protected void setWriteSchemaForDeletes(HoodieTableMetaClient metaClient) {
         } else {
           throw new HoodieIOException("Latest commit does not have any schema in commit metadata");
         }
+      } else {
+        LOG.warn("Cannot find last instant in the timeline");
```

Comment:
How about `0 rows are deleted because the table is empty`

Reply:
> How about `0 rows are deleted because the table is empty`

completed